local-cli/eject/eject.js

```diff
@@ -84,6 +129,54 @@ function eject() {
       appName,
       templateOptions
     );
+
+    console.log('Setting up App Icons for Android');
+    const hdpiPath = 'android/app/src/main/res/mipmap-hdpi/ic_launcher.png';
```

Comment:
It looks like this is duplicated with ICON_SIZES below. Can you refactor it so that these strings are not duplicated?